go/test/endtoend/reparent/utils/utils.go

```diff
@@ -76,7 +77,7 @@ func SetupRangeBasedCluster(ctx context.Context, t *testing.T) *cluster.LocalPro
 }
 
 // SetupShardedReparentCluster is used to setup a sharded cluster for testing
-func SetupShardedReparentCluster(t *testing.T, durability string) *cluster.LocalProcessCluster {
+func SetupShardedReparentCluster(t *testing.T, durability string, extraVttabletFlags map[string]string) *cluster.LocalProcessCluster {
```

Comment:
Let’s keep this as a string slice, it will be easier to append and will also maintain the same ExtraArgs type.

Reply:
Fixed via 89060c80363